cmd/root.go

```diff
@@ -43,6 +49,9 @@ or servererror`,
 	}
 )
 
+func init() {
+	codeCmd.PersistentFlags().BoolVar(&print, "print", false, "Prints respective rfc")
```

Comment:
```suggestion
	codeCmd.PersistentFlags().Bool("print", false, "Prints respective rfc")
```
as you're not using `var print bool` from l.21